modules/gitops-release-notes-1-2.adoc

```diff
@@ -0,0 +1,132 @@
+
++
+[source,yaml]
+----
+apiVersion: argoproj.io/v1alpha1
+kind: ArgoCD
+metadata:
+  name: example-argocd
+  labels:
+    example: basic
+spec:
+  sso:
+    provider: keycloak
+  server:
+    route:
+     enabled: true
+----
+* GitOps Operator now enables adding labels to routes in ArgoCD CR.  This feature enables defining hostname in route's labels to support router sharding.  To set labels on a route, add `labels` under the `route` configuration for a server in ArgoCD CR.  Setting labels to `server` (argocd server), `grafana`, and `prometheus` routes are supported.  
++
+.Example ArgoCD CR YAML to set labels on argocd server
+[source,yaml]
+----
+apiVersion: argoproj.io/v1alpha1
+kind: ArgoCD
+metadata:
+  name: example-argocd
+  labels:
+    example: basic
+spec:
+  server:
+    route:
+     enabled: true
+     labels: 
+       key1: value1
+       key2: value2
+----
+* GitOps Operator automatically grants permissions to Argo CD instances to manage resources in target namespaces by applying labels. Users can label the target namespace with label `argocd.argoproj.io/managed-by: <source-namespace>`, where `source-namespace` is the namespace where the argocd instance is deployed in. 
+
+[id="fixed-issues-1-2_{context}"]
+== Fixed issues
+The following issues were resolved in the current release:
+
+* Previously, if a user decides to create additional instances of Argo CD that are managed by the cluster instance of Argo CD in the `openshift-gitops` namespace, the ArgoCD Application responsible for the new Argo CD instance now gets stuck in `OutOfSync` status. link:https://issues.redhat.com/browse/GITOPS-1025[GITOPS-1025]
+
+[id="known-issues-1-2_{context}"]
+== Known issues
+These are the known issues in {gitops-title} 1.2:
+
+* `argocd.argoproj.io/managed-by` labels in target namespaces are not removed, when Argo CD instance is deleted from the source namespace.  link:https://issues.redhat.com/browse/GITOPS-1228[GITOPS-1228]
+
+* When upgrade from {gitops-title} `v1.1.2` to `v1.2`, resource quota has been enabled in the `openshift-gitops` namespace. It can affect out-of-band workloads deployed manually and workloads deployed by the default Argo CD instance in the `openshift-gitops` namespace.   These workloads must be configured with resource requests/limits.  The resource quota in the `openshift-gitops` namespace may need to be increased if there are any additional workloads.
+
+ Current ResourceQuota for openshift-gitops namespace.
+[cols="1,1",options="header"]
+|===
+| *Resource* | *Requests* | *Limits* 
+| CPU
+| 6688m
+| 13750m
+| Memory
+| 4544Mi
+| 9070Mi
+|===
+ 
+ You can use the below command to update the CPU limits.
+
+[source,shell]
+----
+$ oc patch resourcequota openshift-gitops-compute-resources -n openshift-gitops --type='json' -p='[{"op": "replace", "path": "/spec/hard/limits.cpu", "value":"9000m"}]'
+----
+
```

Comment:
Add a '+' sign on this line so that the next line is aligned to the previous paragraph and renders correctly.